tests/UI/campaigns/utils/globals.js

```diff
@@ -47,6 +47,6 @@ global.TAKE_SCREENSHOT_AFTER_FAIL = process.env.TAKE_SCREENSHOT_AFTER_FAIL || fa
 
 global.maildevConfig = {
   smtpPort: process.env.SMTP_PORT || '1025',
-  smtpServer: process.env.SMTP_SERVER || 'localhost',
+  smtpServer: process.env.SMTP_SERVER || '172.20.0.4',
```

Comment:
what's this IP? :thinking:

Reply:
If I'm not wrong, it's the IP shared between PrestaShop and tests Containers.
We start the maildev server on tests, and to make the shop able to communicate with it, we should use the ip and not localhost.
You could see the config [here](https://github.com/PrestaShop/PrestaShop/blob/develop/tests/UI/docker-compose.tests.yml#L3-L7).